### PR TITLE
Initial commit

### PR DESCRIPTION
**Pull request by taniaprocha**
- Add gatsby configurations
- Add some initial components

### CODE REVIEW

**Review by RafaelFerreiraTVD — APPROVED** [review]
Well done 😃

File: package.json [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+    "url": "https://github.com/Tech4covid19/trackovid-landing-page"
+  },
+  "scripts": {
+    "dev": "gatsby develop",
+    "build": "gatsby build",
+    "lint": "eslint src",
+    "lint-styles": "stylelint \"src/**/*.css\" \"src/**/*.scss\""
+  },
+  "dependencies": {
+    "axios": "^0.19.2",
```
Comment (ruisaraiva19) [inline package.json]: We should pin all packages versions. And to keep dependencies updated, we can add the renovate bot to do that for us 🙂 .

File: src/styles/breakpoints.css [hunk 2]
```diff
@@ -0,0 +1,32 @@
+$Breakpoint-mobile: 986px;
```
Comment (ruisaraiva19) [inline src/styles/breakpoints.css]: Can we use either `snake_case` or `kebab-case`?

File: src/styles/breakpoints.css [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+$Breakpoint-desktop: 1268px;
+
+@mixin Breakpoint-mobileOnly {
```
Comment (ruisaraiva19) [inline src/styles/breakpoints.css]: Same for mixin names.

**Review by ruisaraiva19 — CHANGES_REQUESTED** [review]
Looks good. I just left a couple of comments. Also, we should use lower case file names, just for consistency reasons and cross-OS compatibility issues.

File: package.json (hunk 1 above)
Comment (taniaprocha) [reply package.json]: Can you do that after this commit?

Comment (ruisaraiva19) [reply package.json]: Yes.

File: src/styles/breakpoints.css (hunk 2 above)
Comment (taniaprocha) [reply src/styles/breakpoints.css]: ✅

**Comment by taniaprocha** [comment]
@ruisaraiva19 "lower case file names" are you talking about folders or files? because files are `index.js` or `index.module.css`

**Comment by ruisaraiva19** [comment]
> are you talking about

Both folders and files. I'm almost ready to push a new commit.

**Comment by ruisaraiva19** [comment]
@taniaprocha I just pushed a couple of commits to improve setup consistency.

**Pull request merged**